src/api/v1/auth.ts

```
@@ -0,0 +1,330 @@
+import { Elysia, t } from "elysia";
+import { userService } from "../../pages/user";
+import db from "../../db/db";
+import { User } from "../../db/types";
+import { ACCOUNT_REGISTRATION, ALLOW_UNAUTHENTICATED } from "../../helpers/env";
```

ESLint

[error] 'ALLOW_UNAUTHENTICATED' is defined but never used.

(@typescript-eslint/no-unused-vars)